subsys/mgmt/mcumgr/transport/smp_udp.c

```diff
@@ -9,17 +10,14 @@
  * @brief UDP transport for the mcumgr SMP protocol.
  */
 
-#include <zephyr/kernel.h>
```

Comment:
This should be needed, it's only still building because init.h also currently includes this

Reply:
Yes, so it is redundant to add it twice.